TensorFlow Lite label_image fails to cross-compile with cmake

### Issue type

Bug

### Have you reproduced the bug with TensorFlow Nightly?

No

### Source

source

### TensorFlow version

v2.18.0-rc0

### Custom code

No

### OS platform and distribution

WSL Ubuntu 20.04

### Mobile device

_No response_

### Python version

_No response_

### Bazel version

_No response_

### GCC/compiler version

clang 17.0.2

### CUDA/cuDNN version

_No response_

### GPU model and memory

_No response_

### Current behavior?

I got the error

> [riscv64-linux/bin/ld: cannot find -lprotobuf: No such file or directory](riscv64-unknown-linux-gnu-clang++: error: linker command failed with exit code 1 (use -v to see invocation))


### Standalone code to reproduce the issue

```shell
$ mkdir workspace && cd workspace
$ mkdir flatc_build && mkdir tflite_build

$ wget https://raw.githubusercontent.com/google/XNNPACK/refs/heads/master/cmake/riscv64.toolchain
$ wget https://github.com/riscv-collab/riscv-gnu-toolchain/releases/download/2023.11.20/riscv64-glibc-ubuntu-20.04-llvm-nightly-2023.11.20-nightly.tar.gz
$ tar zxvf riscv64-glibc-ubuntu-20.04-llvm-nightly-2023.11.20-nightly.tar.gz

$ git clone https://github.com/tensorflow/tensorflow
$ cmake -B ./flatc_build -S ./tensorflow/tensorflow/lite/tools/cmake/native_tools/flatbuffers
$ cmake --build ./flatc_build -j6

$ cmake -B ./tflite_build \
-S ./tensorflow/tensorflow/lite/ \
-DCMAKE_TOOLCHAIN_FILE=$(pwd)/riscv64.toolchain \
-DRISCV_TOOLCHAIN_ROOT=$(pwd)/riscv \
-DRISCV_QEMU_ROOT=$(pwd)/riscv \
-DCMAKE_BUILD_TYPE=Release \
-DTFLITE_HOST_TOOLS_DIR=./flatc_build/flatbuffers-flatc/bin \
-DTFLITE_KERNEL_TEST=OFF

$ cmake --build ./tflite_build -j6
$ cmake --build ./tflite_build -t label_image -j6
```


### Relevant log output

```shell
[100%] Linking CXX executable label_image
/home/xxx/workspace/tflite_build/../riscv/bin/riscv64-unknown-linux-gnu-ld: cannot find -lprotobuf: No such file or directory
riscv64-unknown-linux-gnu-clang++: error: linker command failed with exit code 1 (use -v to see invocation)
```

--- a/tensorflow/lite/examples/label_image/CMakeLists.txt
+++ b/tensorflow/lite/examples/label_image/CMakeLists.txt
@@ -84,5 +84,5 @@ target_compile_options(label_image
 target_link_libraries(label_image
   tensorflow-lite
   profiling_info_proto
-  protobuf
+  libprotobuf
 )

